org.adempiere.base/src/org/compiere/print/DataEngine.java

```diff
@@ -86,18 +86,20 @@
 	 */
 	public DataEngine (Language language)
 	{
-		this(language, null);
+		this(language, null, 0);
 	}	//	DataEngine
 	
 	/**
 	 *	Constructor
 	 *	@param language Language of the data (for translation)
-	 *  @param trxName
+	 *	@param trxName
+	 *	@param windowNo
 	 */
-	public DataEngine (Language language, String trxName){
+	public DataEngine (Language language, String trxName, int windowNo){
```

Comment:
found another backward compatibility issue - please keep the existing public DataEngine (Language language, String trxName) constructor.

Reply:
This resolved